src/Component/addBook.js

```diff
@@ -1,14 +1,45 @@
-import React from 'react';
-
-const BookAdd = () => (
-  <div>
-    <h3>Register new books</h3>
-    <form className="add-form">
-      <input type="text" name="title" placeholder="Book title.." />
-      <input type="text" name="author" placeholder="Author.." />
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const BookAdd = () => {
+  const dispatch = useDispatch();
+  const [book, changeBook] = useState({ title: '', author: '' });
+
+  const changeTitle = (e) => {
+    changeBook({
+      ...book,
+      title: e.target.value,
+    });
+  };
+
+  const changeAuthor = (e) => {
+    changeBook({
+      ...book,
+      author: e.target.value,
+    });
+  };
+
+  const addEventBookHandler = (e) => {
+    e.preventDefault();
+    const { title, author } = e.target.elements;
+    const newBookObj = {
+      title: title.value,
+      author: author.value,
+    };
+    dispatch(addBook(newBookObj));
+  };
```

Comment:
- [ ] [optional] I would suggest resetting the form fields after adding a book to this store, This will improve the user's experience. You can use the javascript function to do that `form.reset()`.